addons/website/models/ir_ui_view.py

```diff
@@ -29,6 +29,19 @@ def _compute_first_page_id(self):
         for view in self:
             view.first_page_id = self.env['website.page'].search([('view_id', '=', view.id)], limit=1)
 
+    @api.multi
+    def name_get(self):
+        if not self._context.get('display_website') and not self.env.user.has_group('website.group_multi_website'):
```

Comment:
@JKE-be Change done, calling super if no context and no multi website enable.
It means now all view m2o select will show the website id when multi website enable.